ugs-core/src/com/willwinder/universalgcodesender/model/GUIBackend.java

```diff
@@ -720,40 +716,38 @@ public void commandSent(GcodeCommand command) {
 
     @Override
     public void commandComplete(GcodeCommand command) {
-        if (command.isError()) {
-            if (this.sendingFile && !this.isPaused()) {
-                try {
-                    this.pauseResume();
-                } catch (Exception e) {
-                    GUIHelpers.displayErrorDialog(e.getLocalizedMessage());
-                }
+        if (command.isError() && this.controller.isStreaming() && !this.isPaused()) {
+            try {
+                this.pauseResume();
+            } catch (Exception e) {
+                GUIHelpers.displayErrorDialog(e.getLocalizedMessage());
+            }
 
-                /*
```

Comment:
This commented out code can be deleted, it caused a ton of problems there's no way we'll want to re-enable it.